wpcom-vip-two-factor/sms-provider.php

```diff
@@ -79,14 +79,25 @@ public function delete_token( $user_id ) {
 		delete_user_meta( $user_id, self::TOKEN_META_KEY );
 	}
 
+	/**
+	 * Standardize the format of the SMS message
+	 *
+	 * @param string $token User token.
+	 */
+	public function format_sms_message( $token ) {
+		$message = $token . " is your " . $_SERVER['HTTP_HOST'] . " verification code.\r\n" . "@" . $_SERVER['HTTP_HOST'] . " #" . $token;
```

Comment:
You could break this across 2 lines

Reply:
Might be better/easier to use sprintf like in here:
https://github.com/Automattic/vip-go-mu-plugins/blob/master/cache/class-vary-cache.php#L48

Also, not sure if this needs it's own function, but I think maybe you did this to make testing easier?

Reply:
Broke the lines up!
Yes, it was my attempt to isolate the code for testing, but I opted to recombine the code to spy on the function instead like we discussed!

Reply:
This looks good, in context of our testing challenges today, but any thoughts on using `sprintf` here?

Reply:
Changed to using `sprintf()` for standardization across the codebase :)